engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -243,6 +223,12 @@ class WorkflowManagerActor(params: WorkflowManagerActorParams)
   when (Done) { FSM.NullFunction }
 
   whenUnhandled {
+    case Event(AbortWorkflowsCommand(ids), stateData) =>
+      for {
+        id <- ids
+        actor <- stateData.workflows.get(id)
+      } yield actor ! WorkflowActor.AbortWorkflowCommand
+      stay()
```

Comment:
@mcovarr is this logic enough now ? It seems simpler compared to what was there before but was deleted https://github.com/broadinstitute/cromwell/pull/4401/files#diff-916acc6cffb855ee13688ea50412095fL172